app/views/users/new.html.haml

```diff
@@ -1,17 +1,39 @@
+= content_for :head do
+  %script(type="text/javascript" src="https://js.stripe.com/v2/")
+  %script(type="text/javascript")
+    Stripe.setPublishableKey('pk_test_7IlBHzY9NLZhj3TW4grIrSXZ');
```

Comment:
I would move the public key under an env variable, because if you'd launch a real app then you would use test keys for development, test and staging and real key for production. This hardcoded key would not work in production though.